exec-tests

```diff
@@ -238,9 +238,11 @@ if [[ "${major}" == "all" || "${major}" == "server" ]]; then
     if [[ "${subtst}" == "functional" && ${rc} -eq 0 ]]; then
         server_arg=${1}
         shift
+        keep_datasets=${1}
+        shift
         posargs="${@}"
```

Comment:
If I specify a positional argument _without_ specifying `KEEP_DATASETS`, isn't line 241 going to eat my argument (and do the wrong thing with it)?

Reply:
Well, yes; like the server address, it's a required parameter. This only applies to functional test runs. There are ways we could get around this, but is this really worth worrying about? It simply means that if you ever want to pass true `pytest` positional arguments to a functional test run through `exec-tests` you need to include the "keep datasets" parameter.